bot/cogs/site.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, bot: Bot):
     @site_group.command(name="help")
     async def site_help(self, ctx: Context) -> None:
         """Info about the site's Getting Help page."""
-        url = f"{PAGES_URL}/asking-good-questions"
+        url = f"{PAGES_URL}resources/guides/asking-good-questions"
```

Comment:
```suggestion
        url = f"{PAGES_URL}/resources/guides/asking-good-questions"
```
I'm a dumb, forgot the slash.